internal/daemon/daemon.go

```
@@ -168,6 +202,49 @@ func (d *Daemon) Start(ctx context.Context) error {
+
+	// Optional workspace Unix socket for clamshell container bridge access.
+	// The workspace directory is bind-mounted into clamshell containers so the
+	// bridge can reach the daemon via /workspace/.belayer/daemon.sock without
+	// any network proxying.
+	if d.config.WorkspaceSockPath != "" {
+		if err := os.MkdirAll(filepath.Dir(d.config.WorkspaceSockPath), 0o755); err != nil {
+			ln.Close()
+			return fmt.Errorf("daemon: create workspace sock dir: %w", err)
+		}
+		os.Remove(d.config.WorkspaceSockPath)
+		wsLn, err := net.Listen("unix", d.config.WorkspaceSockPath)
```

golangci-lint (2.11.4)

[error] net.Listen must not be called. use (*net.ListenConfig).Listen

(noctx)